src/routes/docs/tutorials/nuxt/step-6/+page.markdoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: tutorial
+title: Add database
+description: Add databases and queries for ideas in your Nuxt project.
+step: 6
+---
+
+In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store the ideas.
+
+{% only_dark %}
+![Create project screen](/images/docs/tutorials/dark/idea-tracker-collection.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/tutorials/idea-tracker-collection.png)
+{% /only_light %}
+
+Create a new collection with the following attributes:
+| Field       | Type   | Required |
+|-------------|--------|----------|
+| userId      | String | Yes      |
+| title       | String | Yes      |
+| description | String | No       |
+
+Change the collection's permissions in the settings to give access.
+First, add the role "Any" and check the `READ` box.
```

Comment:
Applies to all, in general in docs, we don't use quotes to specify things we refer to in UI.

For anything you're quoting from UI, use **bold**.

For anything you quote from a file or is copy-able, use a `code-tags`